packages/dds/tree/src/forest/cursorLegacy.ts

```diff
@@ -64,9 +64,8 @@ export interface ITreeCursor<TResult = TreeNavigationResult> {
     readonly type: TreeType;
 
     /**
-     * @returns the keys of the currently selected node.
+     * @returns the keys of the currently selected node. Only non-empty fields will be included.
```

Comment:
updating the legacy cursor spec and its implementations is not really needed (it's a problem we could have solved by just finishing the API migration). Since you did the fix we might as well merge it though.